lib/matplotlib/table.py

```diff
@@ -453,19 +476,28 @@ def _auto_set_font_size(self, renderer):
 
         if len(self._cells) == 0:
             return
-        fontsize = list(six.itervalues(self._cells))[0].get_fontsize()
-        cells = []
+
+        fontsize = None
+        grow = self._bbox
```

Comment:
probably better to do `grow = self._box is not None` instead of relying on the truethiness of bbox objects.